src/bucket/BucketManagerImpl.cpp

```diff
@@ -828,6 +830,7 @@ BucketManagerImpl::addBatch(Application& app, uint32_t currLedger,
                                   deadEntries.size());
     mBucketList->addBatch(app, currLedger, currLedgerProtocol, initEntries,
                           liveEntries, deadEntries);
+    mBucketListSizeMeter.Mark(mBucketList->getSize());
```

Comment:
I find this metric a bit confusing. Meters are good for "rate of something happening per desired unit of time". Here we update the meter with full BL size every ledger, but the meter's default unit of time is seconds, so the resulting rate you'll see will be bytes per second (BL size divided by 5 roughly) aggregated over several ledgers. So I'm not sure how to reason about it. 

You probably want to use a counter that you set once a ledger to get precise BL size, then our metrics exporter can properly scrape it and plot the size changes overtime